src/test/java/org/folio/permstest/TestUtil.java

```diff
@@ -72,8 +59,8 @@ public JsonObject getJson() {
       .putHeader("content-type", CONTENT_TYPE_JSON)
       .putHeader("accept", CONTENT_TYPE_JSON);
     if (headers != null) {
-      for (Map.Entry entry : headers.entries()) {
-        request.putHeader((String) entry.getKey(), (String) entry.getValue());
+      for (Map.Entry<String, String> entry : headers.entries()) {
+        request.putHeader(entry.getKey(), entry.getValue());
```

Comment:
Can you use HttpRequest#putHeaders(MultiMap) ?
https://github.com/vert-x3/vertx-web/blob/4.0.0/vertx-web-client/src/main/java/io/vertx/ext/web/client/impl/HttpRequestImpl.java#L160-L164

Reply:
yes,